frontend/src/Image/ImageRef.js

```diff
@@ -4,7 +4,7 @@ export default function ImageRef(src, missing_url) {
   return new Promise( (resolve, reject) => {
     // check for undefined, url, image mongo id, file
     if ( typeof src === 'undefined' || src === null) {
```

Comment:
Can't we just do `!src` I mean it's just semantics It doesn't make a difference tbf...